docs/03-data-model/json-field-typing-inventory.md

```
@@ -1,90 +1,380 @@
+| Field | Bronze inferred | Silver Pandera | Silver PyArrow | Gold contract |
+| --- | --- | --- | --- | --- |
+| `_ingestion_ts` | `unknown` (nullable) | `str` (not-null) | `string` (nullable) | — |
+| `_lookup_method` | `unknown` (nullable) | `str` (not-null) | `string` (nullable) | `str` (not-null) |
+| `_original_id` | `unknown` (nullable) | `str` (nullable) | `string` (nullable) | `str` (nullable) |
+| `_run_id` | `unknown` (nullable) | `str` (not-null) | `string` (nullable) | — |
+| `_run_type` | `unknown` (nullable) | `str` (not-null) | `string` (nullable) | — |
+| `_source` | `unknown` (nullable) | — | `string` (nullable) | `str` (not-null) |
+| `_source_batch_id` | `unknown` (nullable) | `str` (nullable) | `string` (nullable) | — |
+| `_state` | `unknown` (nullable) | `str` (not-null) | `string` (not-null) | — |
+| `abstract` | `unknown` (nullable) | `str` (nullable) | `string` (nullable) | `str` (nullable) |
+| `accession` | `null|string` (nullable) | `str` (not-null) | `string` (not-null) | `str` (not-null) |
```

markdownlint-cli2 (0.23.1)

[warning] Table column count
Expected: 5; Actual: 6; Too many cells, extra data will be missing

(MD056, table-column-count)